sgt-back-end/index.js

```diff
@@ -0,0 +1,159 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+
+  const sql = `
+    select "gradeId",
+           "name",
+           "course",
+           "score",
+           "createdAt"
+      from "grades"
+      `;
+  db.query(sql)
+    .then(result => {
+      const grade = result;
+      if (!grade) {
+        res.status(404).json({
+          Error: 'Cannot find grade'
+        });
+      } else {
+        res.json(grade.rows);
+      }
```

Comment:
Here you don't need to check anything, if there are no grades it will send back an empty array which is fine. So you can do:
```JAVASCRIPT
const grades = result.rows;

res.json(grades);
```